CMakeLists.txt

```diff
@@ -283,21 +289,30 @@ elseif(NOT WIN32) # TODO(RSDK-10366): Currently, rust_utils is not published for
   if(NOT lvru_status_code EQUAL 0)
     message(FATAL_ERROR "No local viam_rust_utils found and failed to download: ${lvru_status_string}")
   endif()
-
+else()
+    message(WARNING "Currently running on Windows with no rust-utils file. Module code should work as expected, but client code may fail unexpectedly.")
 endif()
 
-# TODO(RSDK-10366): Currently, rust_utils is not published for windows, so don't even declare the library
-if (NOT WIN32)
+if (NOT WIN32) # build `SHARED` on unix-based systems
   add_library(viam_rust_utils SHARED IMPORTED)
-
   target_link_directories(viam_rust_utils
     INTERFACE
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
     "$<INSTALL_INTERFACE:${CMAKE_INSTALL_LIBDIR}>"
   )
+  set_property(TARGET viam_rust_utils PROPERTY IMPORTED_LOCATION ${viam_rust_utils_file})
 
+elseif (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "AMD64") # build `STATIC` for windows
+  add_library(viam_rust_utils STATIC IMPORTED)
+  target_link_directories(viam_rust_utils
+    INTERFACE
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>"
```

Comment:
Apologies: I've changed my mind here, I think you still do need `INSTALL_INTERFACE` here for the `BUILD_SHARED_LIBS=OFF` case.